__tests__/Hospital-Detail /ContactCard.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
```

Comment:
change the name Hospital-Detail to hospital-detail folder names should be in a small case.